src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -31,8 +29,12 @@
     private String projectUrl;
 
     @DataBoundConstructor
-    public GithubProjectProperty(String projectUrl) {
-        this.projectUrl = new GithubUrl(projectUrl).baseUrl();
+    public GithubProjectProperty(String url) {
+        this.projectUrl = new GithubUrl(url).baseUrl();
+    }
+
+    public String getUrl() {
```

Comment:
Because the databinding control as well as non-UI tools like `DescribableHelper` expect to call the bean getter directly.